packages/cloud_firestore/cloud_firestore/darwin/Classes/UserAgent.h

```diff
@@ -0,0 +1,3 @@
+// Generated file, do not edit
```

Comment:
You shouldn't need to check this file in, it's generated automatically in packages/cloud_firestore/cloud_firestore/macos/cloud_firestore.podspec

Reply:
Oh right. Removed them.